examples/lb3-application/lb3app/test/authentication.js

```diff
@@ -0,0 +1,94 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+const lb3App = require('../server/server');
+const supertest = require('@loopback/testlab').supertest;
+const assert = require('assert');
+const ExpressServer = require('../../dist/server').ExpressServer;
+require('should');
+
+let app;
+
+function request(verb, url) {
+  // use the original app's server
+  return supertest(app.server)
+    [verb](url)
+    .set('Content-Type', 'application/json')
+    .set('Accept', 'application/json')
+    .expect('Content-Type', /json/);
+}
+
+describe('LoopBack 3 authentication', function () {
+  before(async function () {
+    app = new ExpressServer();
+    await app.boot();
+    await app.start();
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  context('authentication', () => {
+    let User;
+
+    before(() => {
+      User = lb3App.models.User;
+    });
+
+    it('creates a User and logs them in and out', function (done) {
+      User.create({email: 'new@email.com', password: 'L00pBack!'}, function (
+        err,
+        user,
+      ) {
+        assert.equal(user.email, 'new@email.com');
+        User.login(
```

Comment:
Considering that the test name says "it creates a user and logs them in and out", I think the test should call `User.create`, `User.login` and `User.logout` via REST API?

I apologize if this is contradicting my previous comments, I guess I didn't realize what exactly is this test about 🙈